components/ai_chat/browser/ai_chat_tab_helper.cc

```diff
@@ -360,6 +365,65 @@ void AIChatTabHelper::MakeAPIRequestWithConversationHistoryUpdate(
            {prompt_segment_history, question_part}, nullptr),
        ai_chat::kAIPrompt, " <response>\n"});
 
+  return prompt;
+}
+
+std::string AIChatTabHelper::BuildLlamaPrompt(
+    const mojom::ConversationTurn& turn,
+    bool is_suggested_question) {
+  std::string question_part;
+  // TODO(petemill): Tokenize the summary question so that we
+  // don't have to do this weird substitution.
+  if (turn.text == "Summarize this video") {
+    question_part =
+        l10n_util::GetStringUTF8(IDS_AI_CHAT_QUESTION_SUMMARIZE_VIDEO);
+  } else {
+    question_part = turn.text;
+  }
+
```

Comment:
This part seems the same for Claude and Brave. Perhaps we somehow move that part to an anonymous function, or inside `MakeAPIRequestWithConversationHistoryUpdate` before the split?

Reply:
I updated so that these are different for Claude and LLama  -- Llama can't really handle timestamps, so I updated it's video summary prompt to exclude that part.